### PR TITLE
Gal/run 2662 add tag and trigger operator update version

### PR DESCRIPTION
**Pull request by galbenyair**

### CODE REVIEW

**Review by omer-dayan — COMMENTED** [review]
Loved the change. Lets do it on 'deploy-staging' to. Istead of pushing it prod env, it will push it to staging

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,6 +13,29 @@ commands:
...
+            echo "Updating mig-parted new version number $TAG in runai-operator..."
+            cd ~
+            git config --global -l
+            git config --global user.name "circleci-runai"
+            git config --global user.email "circleci@run.ai"
+            git clone https://github.com/run-ai/runai-operator.git
+            cd runai-operator/subcharts/mig-parted
+            echo $TAG
+            yq e -i '.image.tag = strenv(TAG)' values.yaml
+            cat values.yaml
```
Comment (gshaibi) [inline .circleci/config.yml:35]: should we remove it?

**Review by gshaibi — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (galbenyair) [reply .circleci/config.yml:35]: which part?

Comment (omer-dayan) [reply .circleci/config.yml:35]: The cat commant

File: .circleci/config.yml [hunk 2]
```diff
@@ -24,17 +47,18 @@ jobs:
...
+        name: Build And Push to google cloud staging registry with revision tag
         command: cd deployments/gpu-operator && make
         environment:
           ENVIRONMENT: staging
           VERSION: <<pipeline.git.revision>>
     - run:
-        name: deploy
-        command: pwd
+        name: Build And Push to google cloud staging registry with 'latest' tag
+        command: cd deployments/gpu-operator && make
```
Comment (omer-dayan) [inline .circleci/config.yml:57]: We dont need to run the whole make again, it make the build stage as well, which is redundant. Today our Makefile has a target that also build and push. Lets change it to 2 different targets. One that build and one that push and then we would be able to build once and push twice

**Review by omer-dayan — APPROVED** [review]

**Pull request merged**